src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -94,12 +170,13 @@ export function transformVAOSAppointment(appt) {
     practitioners: appt.practitioners,
     vaos: {
       isVideo,
+      isPastAppointment: isPast,
       appointmentType: getAppointmentType(appt),
       isCommunityCare: isCC,
       isExpressCare: false,
       requestVisitType: getTypeOfVisit(appt.kind),
       apiData: appt,
-      // TODO missing data: isPastAppointment, isCOVIDVaccine, timeZone
+      // TODO missing data: isCOVIDVaccine, timeZone
```

Comment:
We won't ever have `timeZone` here, so we can probably ignore that. We could set `isCOVIDVaccine` to be `appt.serviceType === COVID_VACCINE_ID` (that constant should be in the constants file).